src/components/SecondaryPanes/Breakpoints/tests/Breakpoint.spec.js

```diff
@@ -50,6 +55,46 @@ describe("Breakpoint", () => {
     });
     expect(component).toMatchSnapshot();
   });
+
+  it("gets breakpoints prop from Connect version #1", () => {
+    const store = createStore();
+    const component = shallow(<Breakpoint />, { context: { store } });
+    expect(component.prop("breakpoints")).toBe(
+      store.getState().breakpoints.breakpoints
+    );
+  });
```

Comment:
I think i like #1 the most.

It would be nice to change the test a bit so that instead of asserting that we have the same breakpoints, we assert that the render was successful and that the user can infact click an option like `disable`.

This might be better as a mochitest, at which point, i'd be happy to create one which is similar to `breakpoints-cond` which also uses a context menu i believe